ed_sensor_integration/include/ed/kinect/fitter.h

```diff
@@ -239,6 +252,7 @@ class Fitter
     ed::models::ModelLoader model_loader_;
 
     uint nr_data_points_;
+    bool configured_ = false;
```

Comment:
Still not sure what I think of this configured thing. Can we not provide all the required data at construction?

Reply:
I can imagine situations in the future where one would want to reconfigure the fitter depending on the object to be fitted. e.g. if it is large in the image or small.
The issue with the constructors you mentioned also relates to this. Since we configure the fitter when we receive the first camera image we do not have the necessary parameter to configure the fitter on construction. If we were to add these parameters we will always assume that it matches the camera info we use. In practice only the default constructor is used. The others might provide some shorthand utility.